pkg/component/extensions/operatingsystemconfig/operatingsystemconfig_test.go

```diff
@@ -782,6 +787,65 @@ var _ = Describe("OperatingSystemConfig", func() {
 					Expect(actual).To(Equal(obj))
 				}
 			})
+
+			Context("CABundle generation", func() {
```

Comment:
Strictly speaking, we are not generating anything.

```suggestion
			Context("CA bundle computation", func() 
```